src/components/buttonMore/button-more.js

```diff
@@ -0,0 +1,49 @@
+import template from "./button-more.html";
+import { MDCMenu } from "@material/menu";
+
+export class ButtonMoreComponent {
+  constructor(mountPoint, position) {
```

Comment:
please pass `position` inside a `prop` to component

`constructor(mountPoint, props)`